src/pages/projects.html

```diff
@@ -102,10 +102,10 @@
 						<div class="list__item">
 							<div class="tile type_project">
 								<div class="tile__icon">
-									<div class="icon size_96x96 name_express-ui5"></div>
+									<div class="icon size_96x96 name_easy-ui5"></div>
 								</div>
-								<div class="tile__title"><a href="https://github.com/handy-man/Expressui5" class="link" rel="noopener" target="_blank">expressui5</a></div>
-								<div class="tile__text">CLI tool to create UI5 projects quickly from developer-maintained templates.</div>
+								<div class="tile__title"><a href="https://github.com/SAP/generator-easy-ui5" class="link" rel="noopener" target="_blank">Easy UI5</a></div>
+								<div class="tile__text">Generator for OpenUI5-based web-apps which use the official UI5 tooling and support deployment targets such as the SAP Cloud Platform.</div>
```

Comment:
Suggestion to shorten a bit:
Generator for OpenUI5 apps using the official UI5 tooling and supporting deployment to SAP Cloud Platform.

Why:
The texts for the other projects are also very short on this page and focus on the key aspects. You may add the additional info in the table below or in your GitHub readme.

Reply:
I like Michael's suggestion. No further comments. ;-)